shared/src/metabase/mbql/schema.cljc

```diff
@@ -462,8 +468,20 @@
 
 (def ^:private aggregations #{:sum :avg :stddev :var :median :percentile :min :max :cum-count :cum-sum :count-where :sum-where :share :distinct :metric :aggregation-options :count})
 
+(def temporal-extract-functions
+  "Functions to extract components of a date, datetime."
+  #{;; extraction functions (get some component of a given temporal value/column)
+    :temporal-extract
+    ;; SUGAR drivers do not need to implement
+    :get-year :get-quarter :get-month :get-day :get-day-of-week :get-hour :get-minute :get-second})
+
+(def date+time+timezone-functions
+  "Date, time, and timezone related functions."
+  (set/union temporal-extract-functions))
```

Comment:
What does this `set/union` call achieve?

Reply:
I think this anticipates follow-up work that adds to these functions.

Reply:
I tend to add these when needed. You could add a comment in order no to give readers a pause.

Reply:
noted. Sorry about this :)